Frends.OAuth.CreateJWTToken/Frends.OAuth.CreateJWTToken/CreateJWTToken.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Text;
+using Frends.OAuth.CreateJWTToken.Definitions;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Frends.OAuth.CreateJWTToken;
+
+/// <summary>
+/// OAuth operation task.
+/// </summary>
+public class OAuth
+{
+    /// <summary>
+    /// Create OAuth JWTToken.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.OAuth.CreateJWTToken
+    /// </summary>
+    /// <param name="input">Input parameters</param>
+    /// <returns>List { string Token }</returns>
```

Comment:
Please update the documentation that you return an object with `Token` property.